### DISCUSSION TITLE
v2 playful migration attempt for ipfs/ipfs-desktop (electron)

### DISCUSSION THREAD

**Discussion by SgtPooki** [wailsapp/wails#1576] [Show and tell]
I was able to throw together a quick implementation of [ipfs/ipfs-desktop](https://github.com/ipfs/ipfs-desktop) using wails in less than 3 hours. I ran into a number of issues trying to figure out exactly how the `wails dev` hosting of the assets worked.. and ended up serving JS via embed.FS and the static files via `npx serve`. This is probably not the way things are supposed to work, but it's what I ended up with when struggling to get the app to work properly.

P.S. I am not familiar with golang. The most I've done is get a PR ready for some library, and not publish it.

Check it out at https://github.com/SgtPooki/ipfs-desktop-wails

**Comment by leaanthony** [comment]
This is great! Some good feedback in there. Could you please open tickets for what you believe are bugs e.g. open files issue. I'd also like to make the docs easier as it seemed you struggled with that. How can we improve them?
